### PR TITLE
new parameters for tidymodels/embed#213

### PR DESCRIPTION
**Pull request by topepo**
For UMAP steps.

### CODE REVIEW

File: R/param_initial_umap.R [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#' Initialization method for UMAP
+#'
+#' This parameter is the type of initialization for the UMAP coordinates. Can be
+#' one of `"spectral"`, `"normlaplacian"`, `"random"`, `"lvrandom"`,
+#' `"laplacian"`, `"pca"`, `"spca"`, or `"agspectral"`.See [uwot::umap()] for
```
Comment (EmilHvitfeldt) [inline R/param_initial_umap.R:5]:
```suggestion
#' `"laplacian"`, `"pca"`, `"spca"`, or `"agspectral"`. See [uwot::umap()] for
```

File: R/param_target_weight.R [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#' Amount of supervision parameter
+#'
+#' For [uwot::umap()] and [embed::step_umap()], this is a weighting factor
+#' between data topology and target topology. A value of 0.0 weights entirely
+#' on data, a value of 1.0 weights entirely on target. The default of 0.5
+#' balances the weighting equally between data and target.
+#'
```
Comment (EmilHvitfeldt) [inline R/param_target_weight.R:7]: this object should have a `@details` page too

**Review by EmilHvitfeldt — APPROVED** [review]
looks good. 1 request

File: R/param_target_weight.R (hunk 2 above)
Comment (hfrick) [reply R/param_target_weight.R:7]: added in [e0db1be](https://github.com/tidymodels/dials/pull/324/commits/e0db1be81d7f3b37e204db9f6e35e384a04f9825)

**Review by hfrick — APPROVED** [review]

**Comment by hfrick** [comment]
Thanks both! 🙌

**Pull request merged**